Resources/Prototypes/_SpaceCats/Entities/Objects/Fun/pillows.yml

```
@@ -0,0 +1,872 @@
+      collection: ToySqueak
+    damage:
+      types:
+        Blunt: 0
+
+- type: entity
+  name: pillow
+  parent: BaseItem
+  id: Pillow1
+  description: Large pillow with a person printed on top of it's covers. Definitely looks young.
+  components:
+    - type: Sprite
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)